h_periodic/h_beat.py

```diff
@@ -1,9 +1,18 @@
 """Celery beat scheduler process configuration."""
 
+import sys
 from datetime import timedelta
+from os import environ
 
 from celery import Celery
 
+# We don't want periodic tasks for h-qa because h-qa and h-prod share the same
+# database and search index.
+# See https://github.com/hypothesis/playbook/issues/436
+# So we'll set this DISABLE_H_BEAT envvar on h-qa.
+if "DISABLE_H_BEAT" in environ:
```

Comment:
Might also be nice to have this as `environ.get("DISABLE_H_BEAT")` that way we can leave the param in place in AWS and toggle it on and off by adding a value or not.

Reply:
I originally had `asbool(environ.get("DISABLE_H_BEAT"))` which allows you to use `true` and `false` in the envvar value and defaults to `False` if there envvar isn't set. But `asbool` is a Pyramid thing and I removed Pyramid from h-periodic :)

Reply:
Added something similar to Pyramid's `asbool`: https://github.com/hypothesis/h-periodic/blob/259801d5b109b3460b9dc81a9b0cc7fc30cb9e0a/h_periodic/h_beat.py#L10-L21